server.py

```diff
@@ -17,6 +17,9 @@
 from modules.cache import Cache
 from modules.metrics import MetricsHandler
 
+import signal
+import psutil
```

Comment:
we are somewhat following the pep 8 style guide, meaning we should group the imports based on the below link

signal should go in the first group of imports (its a python standard library) and psutil should go in the second group since its a third party import

import sections should also be alphabetical see below for more info

https://peps.python.org/pep-0008/#imports

Reply:
done